src/orion/client/__init__.py

```diff
@@ -279,7 +279,14 @@ def get_experiment(name, version=None, mode="r", storage=None):
 
 
 def workon(
```

Comment:
Since the parallelism is implemented in ExpelimentClient.workon I would avoid modifying this function for now to keep things simple. We can focus on ExpelimentClient.workon for now.

Reply:
do you mean no need to care the parallelism support for this function, and leave ExpelimentClient.workon always with n_jobs=1 from here?

Reply:
Yeah, for now.